x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -45,6 +45,7 @@ export function plugin() {
 
 export { Plugin };
 export * from './plugin';
+export { loadAlerts, loadAlertAggregations } from './application/lib/alert_api';
```

Comment:
@claudiopro Can you use 
```suggestion
export { loadAlerts as loadRules, loadAlertAggregations as loadRuleAggregations} from './application/lib/alert_api';
```

Reply:
What about leaving these as-is and importing them as `loadRules` and `loadRuleAggregations` in RAC instead?